src/main/java/org/spongepowered/api/command/CommandMessageFormatting.java

```diff
@@ -25,17 +25,19 @@
 package org.spongepowered.api.command;
 
 import org.spongepowered.api.text.Text;
-import org.spongepowered.api.text.Texts;
 import org.spongepowered.api.text.format.TextColors;
 
 public class CommandMessageFormatting {
-    public static final Text PIPE_TEXT = Texts.of("|");
-    public static final Text SPACE_TEXT = Texts.of(" ");
-    public static final Text PLUS_TEXT = Texts.of("*");
-    public static final Text NEWLINE_TEXT = Texts.of('\n');
-    public static final Text LT_TEXT = Texts.of("<");
-    public static final Text GT_TEXT = Texts.of(">");
-    public static final Text ELIPSES_TEXT = Texts.of("…");
+
+    private CommandMessageFormatting() {
+    }
+
+    public static final Text PIPE_TEXT = Text.of("|");
+    public static final Text SPACE_TEXT = Text.of(" ");
+    public static final Text STAR_TEXT = Text.of("*");
+    public static final Text LT_TEXT = Text.of("<");
+    public static final Text GT_TEXT = Text.of(">");
+    public static final Text ELLIPSIS_TEXT = Text.of("…");
```

Comment:
I don't see a need for these to be in `Text`, it's a large enough class already.